src/Symfony/Component/HttpClient/MockHttpClient.php

```diff
@@ -54,7 +55,7 @@ public function __construct($responseFactory = null, string $baseUri = null)
      */
     public function request(string $method, string $url, array $options = []): ResponseInterface
     {
-        [$url, $options] = $this->prepareRequest($method, $url, $options, ['base_uri' => $this->baseUri], true);
+        [$url, $options] = self::prepareRequest($method, $url, $options, ['base_uri' => $this->baseUri], true);
```

Comment:
can you please remove this one? it's unrelated to the PR and not needed actually